### PR TITLE
Hire me

### PR DESCRIPTION
**Pull request by MantasDot**

### CODE REVIEW

File: css/components/hireme.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.backgroundHM {
+    background-image: url(/img/HireME.png);
+    background-size: cover;
+    position: fixed;
+    width: 100%;
+    height: 40vh;
```
Comment (GedasPupa) [inline css/components/hireme.css]: reikia pritaikyti responsive'ui.

**Review by GedasPupa — CHANGES_REQUESTED** [review]

**Review by vadimsharanov — APPROVED** [review]
Looks good.

**Review by MantasDot — COMMENTED** [review]
done

File: css/components/hireme.css (hunk 1 above)
Comment (GedasPupa) [reply css/components/hireme.css]: resolved

**Review by GedasPupa — APPROVED** [review]
nice job!

**Pull request merged**